q2_diversity/__init__.py

```diff
@@ -27,5 +27,5 @@
            'core_metrics_phylogenetic', 'core_metrics',
            'filter_distance_matrix', 'mantel', 'alpha_rarefaction',
            'beta_rarefaction', 'procrustes_analysis', 'beta_correlation',
-           'adonis', 'partial_procrustes'
+           'adonis', 'partial_procrustes', 'ignore_missing_samples'
```

Comment:
```suggestion
           'adonis', 'partial_procrustes'
```

Reply:
This isn't needed here since we aren't adding a new method (just exposing a parameter on an existing method).